homeassistant/components/onewire/sensor.py

```diff
@@ -195,12 +224,14 @@
 )
 
 
-def hb_info_from_type(dev_type="std"):
+def get_sensor_types(device_sub_type="std"):
     """Return the proper info array for the device type."""
-    if "std" in dev_type:
+    if "std" in device_sub_type:
         return DEVICE_SENSORS
-    if "HobbyBoard" in dev_type:
+    if "HobbyBoard" in device_sub_type:
         return HOBBYBOARD_EF
+    if "EDS" in device_sub_type:
+        return EDS_SENSORS
```

Comment:
And just add this line at the end:

```suggestion
        return EDS_SENSORS
    return DEVICE_SENSORS
```